iib/workers/tasks/utils.py

```diff
@@ -208,25 +230,25 @@ def serve_index_registry(db_path):
 
 
 @retry(exceptions=IIBError, tries=2, logger=log)
-def serve_index_registry_at_port(db_path, port, max_tries, wait_time):
+def serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
```

Comment:
Careful with refactoring

Reply:
I hope that I am careful. Do you see something that I do not? Maybe I overlooked something...
Thanks